ci/gpu/build.sh

```diff
@@ -51,7 +51,7 @@ gpuci_conda_retry install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvid
       "rmm=${MINOR_VERSION}" \
       "dask-cudf=${MINOR_VERSION}" \
       "dask-cuda=${MINOR_VERSION}" \
-      "ucx-py=${MINOR_VERSION}" \
+      "ucx-py=0.21" \
```

Comment:
This change is available in fedab76. Please merge the latest changes from `branch-21.08` into your PR.

Reply:
done!